src/test/java/com/wcc/platform/controller/AboutControllerTest.java

```diff
@@ -49,4 +49,28 @@ void testInternalError() throws Exception {
                 .andExpect(jsonPath("$.message", is("internal error")))
                 .andExpect(jsonPath("$.details", is("uri=/api/cms/v1/team")));
     }
+
+    @Test
+    void testCollaboratorNotFound() throws Exception {
```

Comment:
It is missing the success case, when we should match the response with success case.

Reply:
I will add the test during the day.

Reply:
Test added